python/tank/platform/application.py

```diff
@@ -161,6 +162,22 @@ def log_error(self, msg):
     def log_exception(self, msg):
         self.engine.log_exception(msg)
 
+    def log_app_metric(self, action):
```

Comment:
I'm thinking this and the corresponding engine method would just bet better as `_log_metric()`.